components/brave_wallet_ui/page/screens/send/send/send.tsx

```diff
@@ -242,6 +244,14 @@ export const Send = (props: Props) => {
     return false
   }, [addressError, toAddress, toAddressOrUrl])
 
+  const showSearchingForAddressIcon = React.useMemo(() => {
+    if ((endsWithAny(allSupportedExtensions, toAddressOrUrl.toLowerCase()) && searchingForDomain) || showEnsOffchainLookupOptions) {
+      setSearchingForAddressIconPosition(document.getElementById(INPUT_WIDTH_ID)?.clientWidth ?? 0)
```

Comment:
side effects belong in useEffect, not useMemo

Reply:
Removed this and moved to `useEffect` hook.